aytests/sles12_minimal_network.list

```diff
@@ -4,4 +4,5 @@ installation_snapshot.sh # snapshot has been created
 no_yast2_packages.sh     # YaST2 packages are not installed
 no_second_stage.sh       # second stage was not run
```

Comment:
if there is no second stage, you're not reproducing the bug correctly (second_stage is default in sle12sp3)

Reply:
Yep, an extra test will be needed because for network we want hosts section.

Reply:
i don't see second stage disabled in the profile, so what case is this test for?

Reply:
I kept the test because covers first stage as you commented. 

Although the bug shows that the /etc/hosts was blanked out during second stage, it could also happens with first stage if we introduce some regression. This test will fail in that case.

Reply:
ups ... overlooked that I'm in another test